### ISSUE TITLE
Compatibility with PhysiCell 1.13.X

### ISSUE THREAD

**Issue by rockdeme** [IGGoncalves/PhysiCOOL#9]
Hey @IGGoncalves!

Just found PhysiCOOL and it seems super nice. Any plans to update the API according to the changes in the current PhysiCell version? It would be great if we could modify the initial conditions in the configuration xml with the parser for example.

Best,
Demeter

**Comment by IGGoncalves** [comment]
Hi @rockdeme!

Thanks for reaching out and showing interest in PhysiCOOL! I'm glad to hear that you find it promising.
Regarding your inquiry, we started adding support for the configuration file parser in the dev branch some time ago, aligning with the changes in the newest PhysiCell versions. I will revise these changes to check if we can incorporate them in a new release soon.

In the meantime, if you have any specific features or improvements in mind that you'd like to see implemented, feel free to share :)

Best regards
Inês

**Comment by rockdeme** [comment]
Hey @IGGoncalves!

Amazing, thanks so much. Maybe a bit out of scope, but I was actually wondering if being able to run multiple simulations in parallel is something that would theoretically be possible with PhysiCOOL?

Best wishes,
Demeter
